### DISCUSSION TITLE
Release plan for March 20, 2023 (8.0.0 - curl 25 years)

### DISCUSSION THREAD

**Discussion by aanchali-progress** [curl/curl#10659] [General]
Hello Everyone,

We are facing a connection issue with Libcurl 7.88.1.
This issue is fixed in the commit https://github.com/curl/curl/commit/233810bb5f6c5e7bedfc10bdd36607b958c0cfe4
Issue : https://github.com/curl/curl/issues/10626
PR : https://github.com/curl/curl/pull/10648

We would like to know if this commit will be there in the upcoming release on 20th March (8.0.0 - curl 25 years - mentioned in the releases plan page).

Since the next version seems to be 8.0.0 which looks like major version upgrade, what would it be like for us to upgrade?
Are we looking into significant changes compared to earlier minor version upgrades ?

Also, as this is issue impacts HTTP Proxy connections, is there any plan to release a hotfix on top of 7.88.1 to limit the impact of adopting major release (8.0)? 

Thanks
Amit

**Comment by bagder** [comment]
Version 8 is **not** going to be a change in behavior, in API nor in ABI.

Version 8 is a way to get back to more sensible version numbers (and to better mark the 25 year anniversary). We have not even added any changes or new features since 7.88.1, so 8.0.0 should work like a 7.88.2 release for all practical purposes.

Upgrading to 8.0.0 will be as smooth and easy as every curl upgrade has been since 2006. We don't break behavior. We don't break ABIs.

Everything that is now in the master branch will be included and shipped in the pending 8.0.0 release. We will not do any other releases before 8.0.0. If you desperately need a fix on top of 7.88.1, then cherry-pick that into your build and run a local version until 8.0.0 ships. (If you cannot do that yourself, you are only a support contract away from getting customized and dedicated private help.)
